tests/test_grammar.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+"""This module has tests for the pvl grammar regexes."""
+
+# Copyright 2019, Ross A. Beyer (rbeyer@seti.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import unittest
+
+from pvl.grammar import PVLGrammar
+
+
+class TestLeapSeconds(unittest.TestCase):
```

Comment:
I thought the library was using pytest for unit tests, can you mix pytest and unittest?

Reply:
The short answer is: yes.

The ``unittest`` library just provides a framework for writing tests.  I happen to prefer it for unit tests, because it has some excellent mocking capabilities.  At any rate, you'll see that the original tests (now in ``tests/test_pvl.py``) are still in ``pytest`` syntax, the newer unit tests that I've written use the ``unittest`` library structure, and the strings in the docs are now properly using doctest.

The beauty is that using pytest as your 'test runner' is pretty omnivorous, and it happily runs all of the tests, no matter the syntax.  So running ``make test`` invokes pytest as the 'test runner' which now runs doctest on the ``docs/`` tree, runs doctest on any doctest-docstrings in the codebase, and then runs the tests in ``tests/`` using the ``pytest`` or ``unittest`` libraries as appropriate.